script/run-bridge-two-vehicles-v2x.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+V2X_PATH=${AUTOWARE_CARLA_ROOT}/external/zenoh_autoware_v2x/
+PYTHON_AGENT_PATH=${AUTOWARE_CARLA_ROOT}/external/zenoh_carla_bridge/carla_agent
+# Log folder
+LOG_PATH=bridge_log/`date '+%Y-%m-%d_%H:%M:%S'`/
+mkdir -p ${LOG_PATH}
+
+# Note bridge should run later because it needs to configure Carla sync setting.
+# Python script will overwrite the settings if bridge run first.
+if [[ ${BRIDGE_TYPE} == "ros2" ]]; then
```

Comment:
We only support ros2 now. You can remove the else branch.
You can take a look at here
https://github.com/evshary/autoware_carla_launch/blob/main/script/run-bridge-two-vehicles.sh